src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,35 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
```

Comment:
Change 'flex-direction' to 'row' for better horizontal layout in the navigation menu.